eacl/target.go

```diff
@@ -77,6 +99,42 @@ func SetTargetECDSAKeys(t *Target, pubs ...*ecdsa.PublicKey) {
 	t.SetBinaryKeys(binKeys)
 }
 
+// SetTargetAccounts sets accounts in Target.
+func SetTargetAccounts(t *Target, accs ...util.Uint160) {
+	binKeys := t.BinaryKeys()
+	ln := len(accs)
+
+	if cap(binKeys) >= ln {
+		binKeys = binKeys[:0]
+	} else {
+		binKeys = make([][]byte, 0, ln)
+	}
+
+	for i := 0; i < ln; i++ {
+		binKeys = append(binKeys, accs[i].BytesBE())
+	}
+
+	t.SetAccounts(binKeys)
+}
+
+// SetTargetUsers sets user ids in Target.
+func SetTargetUsers(t *Target, accs ...user.ID) {
+	binKeys := t.BinaryKeys()
+	ln := len(accs)
+
+	if cap(binKeys) >= ln {
+		binKeys = binKeys[:0]
+	} else {
+		binKeys = make([][]byte, 0, ln)
+	}
+
+	for i := 0; i < ln; i++ {
+		binKeys = append(binKeys, accs[i].WalletBytes())
```

Comment:
again this differs with `SetTargetAccounts`: there u set 20B, here - 25B. Guess u wanted to use `accs[i].GetScriptHash()`